### DISCUSSION TITLE
V11 docs missing lots of pages

### DISCUSSION THREAD

**Discussion by ariaieboy** [spatie/laravel-medialibrary#3463] [General]
lots of pages missing in the main doc website
for example https://spatie.be/docs/laravel-medialibrary/v11/working-with-media-collections/simple-media-collections
https://spatie.be/docs/laravel-medialibrary/v11/working-with-media-collections/defining-media-collections

@freekmurze

**Comment by freekmurze** [comment]
This has been fixed! 👍
